### DISCUSSION TITLE
Simulating improper interactions

### DISCUSSION THREAD

**Discussion by amruthesht** [glotzerlab/hoomd-blue#1807] [Support]
Hello All,

I'm trying to simulate the [DREIDING](https://pubs.acs.org/doi/10.1021/j100389a010) force field with its specific improper interaction. Is there an implementation in hoomd that can be translated into this? LAMMPS has [one](https://docs.lammps.org/improper_umbrella.html); any version available in hoomd would be of great help; thank you!

Best,
Amruthesh

**Comment by joaander** [comment]
I welcome pull requests from the community that add widely used methods. Due to licensing, I am *not* able to accept code that is copied from LAMMPS. 

Users are welcome to extend HOOMD-blue with custom components that implement project-specific methods. The [hoomd-component-template](https://github.com/glotzerlab/hoomd-component-template/) provides everything you need to get started.
